### PR TITLE
feat(TMDB): add presence

### PR DESCRIPTION
**Pull request by keksiqc**
**Describe the changes in this pull request:**
Added `TMDB` presence (again)

<details>
<summary> Proof showing the creation/modification is working as expected </summary>

<br />

<br />

<br />

<br />
and more
</details>

### CODE REVIEW

File: websites/T/TMDB/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (EncryptedDev) [inline websites/T/TMDB/dist/metadata.json:1]: Unformatted

File: websites/T/TMDB/presence.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+			presenceData.details = `Viewing ${document
+				.querySelector(`h2 > [href="${pathname}"]`)
+				.textContent.trim()}`;
+		} else if (pathname.includes("/reviews")) {
+			presenceData.details = `Reading ${document
+				.querySelector(
+					`
+				h2 > [href="${pathname.split("/").slice(0, 3).join("/")}"]
+					`
+				)
```
Comment (EncryptedDev) [inline websites/T/TMDB/presence.ts:52]: Doesn't seem formatted right

File: websites/T/TMDB/presence.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+			presenceData.details = `Viewing ${document
+				.querySelector(`h2 > [href="${pathname}"]`)
+				.textContent.trim()}`;
+		} else if (pathname.includes("/reviews")) {
+			presenceData.state = `Reading ${document
+				.querySelector(
+					`
+				h2 > [href="${pathname.split("/").slice(0, 3).join("/")}"]
+					`
+				)
```
Comment (EncryptedDev) [inline websites/T/TMDB/presence.ts:78]: Ditto

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/T/TMDB/dist/metadata.json (hunk 1 above)
Comment (EncryptedDev) [reply websites/T/TMDB/dist/metadata.json:1]: Pending

Comment (keksiqc) [reply websites/T/TMDB/dist/metadata.json:1]: I don't understand what you mean there

Comment (Slowlife01) [reply websites/T/TMDB/dist/metadata.json:1]: https://github.com/PreMiD/Presences/blob/f06269e9e1d2a0aaaefea08075dc73fb684f2316/websites/T/TMDB/dist/metadata.json

**Comment by EncryptedDev** [comment]
A Russian proofreader will need to approve the strings

File: websites/T/TMDB/presence.ts [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+		presenceData.details = "Viewing leaderboard";
+	else if (pathname.startsWith("/talk"))
+		presenceData.details = "Viewing TMDB support";
+	else if (pathname.startsWith("/search")) {
+		const query = document.querySelector<HTMLInputElement>("input").value;
+		if (query.length > 0) presenceData.details = `Searching for ${query}`;
+		else presenceData.details = "Searching";
+	} else if (pathname.startsWith("/")) presenceData.details = "Browsing TMDB";
+
+	presence.setActivity(presenceData, true);
```
Comment (Slowlife01) [inline websites/T/TMDB/presence.ts:99]:
```suggestion
	presence.setActivity(presenceData);
```

File: websites/T/TMDB/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Keksi♡",
+		"id": "527147599942385674"
+	},
+	"service": "TMDB",
+	"description": {
+		"en": "TMDB: The Movie Database"
```
Comment (ImRodry) [inline websites/T/TMDB/dist/metadata.json:9]:
```suggestion
		"en": "The Movie Database (TMDB) is a popular, user editable database for movies and TV shows."
```

**Review by ImRodry — CHANGES_REQUESTED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Review by ImRodry — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**